keymanapp/keyman
# chore(common): Check in crowdin strings for Waha 📜

This checks in the crowdin strings for the locale `hia` for Waha

- [x] TODO: @sgschantz to check on adding hia (hia-NG?) locale in XCode

## User Testing
For the listed platforms:
1. Load the PR build
2. Set the UI/locale to Waha
3. Verify the UI is Waha
Note: The crowdin translator left some strings in English, so that's out of our control

* **TEST_ANDROID**
1. Verify Waha strings are like the screenshot
![waha](https://user-images.githubusercontent.com/7358010/146155840-d09714a6-8ca1-48f3-81c6-71932a747bc4.png)


* **TEST_IOS** - Wait until Shawn addresses the XCode TODO before testing

* **TEST_LINUX**
1. start Keyman configuration with: `LANGUAGE=hia_NG km-config`

* **TEST_MAC**

* **TEST_WINDOWS**
